synapse/handlers/register.py

```diff
@@ -388,8 +388,8 @@ def get_or_create_user(self, localpart, displayname, duration_seconds):
 
         user = UserID(localpart, self.hs.hostname)
         user_id = user.to_string()
-        auth_handler = self.hs.get_handlers().auth_handler
-        token = auth_handler.generate_short_term_login_token(user_id, duration_seconds)
+        token = self.auth_handler().generate_short_term_login_token(
+            user_id, duration_seconds)
```

Comment:
It's fine, I'll probably clear it up myself since it such a small thing. It was more just an FYI for next time :)